jaxrs-clients/src/main/java/com/palantir/remoting3/jaxrs/AbstractFeignJaxRsClientBuilder.java

```diff
@@ -157,6 +157,9 @@ private static Decoder createDecoder(ObjectMapper objectMapper, ObjectMapper cbo
         // increase default connection pool from 5 @ 5 minutes to 100 @ 10 minutes
         client.connectionPool(new ConnectionPool(100, 10, TimeUnit.MINUTES));
 
+        // logging
+        client.addInterceptor(new HttpLoggingInterceptor(OkhttpSlf4jDebugLogger.INSTANCE));
```

Comment:
I think you can use the default okhttp logger by not passing in any arguments to `HttpLoggingInterceptor`.